rfc/php_technical_committee.wiki

```diff
@@ -0,0 +1,184 @@
+====== PHP RFC: PHP Technical Committee ======
+  * Version: 0.2
+  * Date: 2023-03-03
+  * Author: Jakub Zelenka, bukka@php.net
+  * Status: Draft
+
+===== Introduction =====
+
+PHP uses its voting system to decide about changes to PHP. This was created mainly for user facing changes. It works
+well for that purpose but it is not that effective for deciding about purely technical changes that impact PHP
+internals and extension API's. It is not clear how it should resolve conflicts on the technical basis between
+contributors that sometimes happen.
+
+
+===== Proposal =====
+
+This RFC proposes an introduction of the PHP Technical Committee (TC) that would decide about all technical aspects of
```

Comment:
> all technical aspects

This is a little misleading, as the TC is primarily involved when there are conflicts. They will not be involved for the majority of technical decisions during development.

Reply:
Yeah agreed will clarify it.